### PR TITLE
create new feature.(alert info)

### PR DESCRIPTION
**Pull request by lv7777**
#40 
in tabs-tabs-tabs.

### CODE REVIEW

File: tabs-tabs-tabs/tabs.js [hunk 1]
```diff
@@ -55,5 +55,14 @@ document.addEventListener("click", function(e) {
       chrome.tabs.remove(tab.id);
     });
   }
-
+  
+  else if (e.target.id === "tabs-alertinfo") {
+    callOnActiveTab((tab) => {
+      let props="";
```
Comment (wbamberg) [inline tabs-tabs-tabs/tabs.js]: Please use spaces between operators and operands, like:
`let props = "";`

File: tabs-tabs-tabs/tabs.js [hunk 2]
```diff
@@ -55,5 +55,14 @@ document.addEventListener("click", function(e) {
...
+      for(let a in tab){
```
Comment (wbamberg) [inline tabs-tabs-tabs/tabs.js]: Again:
`for (let a in tab) {`

File: tabs-tabs-tabs/tabs.js [hunk 3]
```diff
@@ -55,5 +55,14 @@ document.addEventListener("click", function(e) {
...
+        props+= a+"="+tab[a]+"\n"
+      }
+      alert(props)
```
Comment (wbamberg) [inline tabs-tabs-tabs/tabs.js]: Semicolon here please:
 `alert(props);`

**Comment by wbamberg** [comment]
Thanks a lot for adding this. I had a few formatting comments, but it seems to work fine on Firefox.

Weirdly, on Chrome, the alert box appears then disappears again immediately. I think this must be because the browser action popup closes as soon as you click the link, and when that closes, its alert is closed as well. If you keep the popup open (by selecting "inspect popup"), then the alert stays open, too.

But then, the popup closes in Firefox too, and the alert stays open there. So maybe that is just a difference in behavior.

I think it would be good for these examples to work in Chrome where that's possible, so it would be good to find a solution for this. We could just log the info to the console (easy), or display it inside the popup (hard). Or maybe use the alert() in the currently active tab, instead of the one in the popup?

**Comment by wbamberg** [comment]
Also, you need to update your branch from master, as it doesn't apply cleanly at the moment (it's missing https://github.com/mdn/webextensions-examples/pull/62/files and https://github.com/mdn/webextensions-examples/commit/8cce7fda06ac8d57bd09fe6caea150825d90dc63).

**Comment by lv7777** [comment]
Thanks for advice and sorry for late.

I just fix formatting problem,and will sending pull request again.

**Comment by lv7777** [comment]
sended.

**Comment by wbamberg** [comment]
Thanks @lv7777 !

**Pull request merged**